ee_tests/src/support/logging.ts

```diff
@@ -0,0 +1,36 @@
+import * as chalk from 'chalk';
+
+export function error(...msg: any[]) {
+    // tslint:disable-next-line:no-console
+    console.error(chalk.red(...msg));
+}
+
+export function spec(...msg: any[]) {
+    // tslint:disable-next-line:no-console
+    console.info(chalk.green(...msg));
+}
```

Comment:
There is something wrong with this... it's very difficult to spot this line in log. Shouldn't `spec` be part of Jasmine reporter?

![screenshot_20180810_101522](https://user-images.githubusercontent.com/445100/43946714-55f11b90-9c86-11e8-9e11-7f717e30c16b.png)

Reply:
It seems that the 'spec' has no color at all. Could you try the new version?